crnlib/crn_image_utils.cpp

```diff
@@ -570,9 +570,9 @@ bool resample_multithreaded(const image_u8& src, image_u8& dst, const resample_p
     return false;
 
   p.m_pSrc_pixels = src_samples.get_ptr();
-  p.m_src_pitch = src_width * resampler_comps * sizeof(float);
+  p.m_src_pitch = (size_t)src_width * (size_t)resampler_comps * sizeof(float);

```

Comment:
Crunch has a maximum image dimension of 4096 so we don't ever need to worry about super-sized memory allocations.